documentation/examples/image/boring_avatar.ts

```diff
@@ -0,0 +1,30 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+pack.addNetworkDomain("source.boringavatars.com");
+
+pack.addFormula({
+  name: "BoringAvatar",
+  description: "Get a boring avatar image.",
+  parameters: [
+    coda.makeParameter({
+      type: coda.ParameterType.Number,
+      name: "Size",
```

Comment:
I've been using all lowercase letters for parameter names. Change this to `size` for consistency.